docs/testing.md

```diff
@@ -177,7 +159,7 @@ After generating the Operator by running `make build-operator`, your newly gener
 is automatically fully integrated into the Submariner CI automation. Simply use
 the `deploytool` flag to the standard `make` commands.
 
-```make ci e2e status=keep deploytool=operator```
```

Comment:
same here, it's for developers, I prefer we keep that by default which is more intuitive to continue with development..